src/http/index.js

```diff
@@ -29,6 +29,22 @@ function hapiInfoToMultiaddr (info) {
   return toMultiaddr(uri)
 }
 
+async function serverCreator (serverAddrs, createServer, hapiInfoToMultiaddr, ipfs) {
```

Comment:
No need to pass hapiInfoToMultiaddr to this function - it’s in scope already

Reply:
Hey when u don't pass that in, it throws an error saying that it is not a function. Not 100% sure why